public/index.pug

```diff
@@ -146,14 +155,12 @@ html(lang="en-US")
                 class="grid-item__button"
                 href=`./icons/${icon.slug}.svg`
                 role="button"
+                data-icon=`${icon.slug}`
                 download
-              ) SVG
-              a(
-                class="grid-item__button"
-                href=`./icons/${icon.slug}.pdf`
-                role="button"
-                download
-              ) PDF
+              )
+                svg(role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" aria-labelledby="download")
```

Comment:
The `aria-labelledby` value should reference an element with the `id="download"` that either has text content or a `title` attribute value. In it's current state (without such an element) it does nothing.

Reply:
Also note that [`aria-labelledby` does not support two elements with the same id](https://web.dev/duplicate-id-aria/).